frontend/src/metabase/css/query_builder.css

```diff
@@ -365,10 +371,6 @@
 
 /* SORT/LIMIT SECTION */
 
-.GuiBuilder-sort-limit {
-  min-width: 0px;
-}
```

Comment:
It feels to me you should move `border-left: 1px solid var(--color-border);` here instead because:
1. The move is consistent with `.GuiBuilder-data` refactoring i.e. you move the rules from the first duplicate selector to the last selector
2. They both are below their own section comment
    1. `/* DATA SECTION */`
    2. `/* SORT/LIMIT SECTION */`